recipe/meta.yaml

```diff
@@ -33,19 +33,19 @@ requirements:
         - xerces-c
         - libnetcdf
         - krb5
-        - libtiff
+        - libtiff 4*
         - libpng
         - openjpeg
         - curl
         - expat
         - zlib  # [not win]
-        - jpeg  # [not win]
+        - jpeg 9*  # [not win]
         - postgresql  # [linux]
         - jasper  # [not win]
         - giflib  # [not win]
         - json-c  # [not win]
         - freexl  # [not win]
-        - xz  # [not win]
+        - xz 5*  # [not win]
```

Comment:
No probs. Just wanted to make sure I understand the problem properly. I think the changes proposed here will help globally across conda-forge.